src/styles/utilities/_variables.scss

```diff
@@ -0,0 +1,57 @@
+@use "sass:map";
+@use './colors' as *;
+
+// colors for mood app
+
+$neutral900: map.get($moodAppColors, neutral900);
```

Comment:
Why is this map function useful? Can't we just use the colors from `colors.scss` file?